src/yb/server/rpc_server.cc

```diff
@@ -57,7 +57,7 @@ using strings::Substitute;
 using std::unique_ptr;
 using std::make_unique;
 
-DEFINE_string(rpc_bind_addresses, "0.0.0.0",
+DEFINE_string(rpc_bind_addresses, "::",
```

Comment:
That would not work if we have only IPv4 addresses available.
If I recall correctly we have to listen IPv4 and IPv6 separately.

Reply:
This definitely works, only does not work when somebody engaged full-on legacy mode (current IP stack completely ripped out of the kernel), it may also not work if current IP sockets don't bind on legacy IP, which is true on OpenBSD and with a kernel setting on Linux. Though if we want separate sockets we need to bind with IPV6ONLY flag on the IPv6 socket and then bind with IPv4 separately.